Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -158,6 +158,13 @@ internal static SearchDefinitions PushInheritedData(SearchDefinitions definition
                         matchExpression.Level = definition.Level;
                     }
 
+                    if (matchExpression.Kind == 0)
```

Comment:
Is it possible to use a enumerated data type instead of fixed integers such as 0? It's not immediately clear to me what is the interpretation when `matchExpression.Kind` is 0.